File: jour_17/jour17.py
while True : 
    try : 
        n = int(input("Entrez la taille du tableau : "))
        if n <= 0 :
            print("Entrez unn nombre positif")
        break
    except ValueError :
        print("Saisie invalid")
tab = []
print("Entrez la valeur de : ")
for i in range(n+1) :
    tab.append(int(input((f"tab[{i}] = "))))
valMIN = tab[0]
for i in range(n + 1) :
    if valMIN > tab[i] : 
        valMIN = tab[i]
valMAX = tab[0]
for i in range(n + 1) : 
    if valMAX < tab[i] : 
        valMAX = tab[i]
print(f"La valeur minimal du tableau est : {valMIN}")
print(f"La valeur maximale du tableau est : {valMAX}\n")

File: jour_17/jour17.c
#include<stdio.h>
int main(){
    int n,result,c;
    do{
        printf("Entrez la taille du tableau : ");
        result = scanf("%d",&n);
        int c;
        while((c = getchar()) != '\n' && c != EOF);
        if (result == 0 || n <= 0){
            printf("Saisie invalid.Veuillez entrez un nombre positif\n");
        }
    }while(result == 0 || n <= 0 );
    int tab[n];
    printf("Entrez la valeur de : \n");
    for (int i = 0; i < n; i++){
        printf("tab[%d] = ",i);
        scanf("%d",&tab[i]);
    }
    int valMIN = tab[0];
    for ( int i = 1; i < n; i++){
        if (valMIN > tab[i])
            valMIN = tab[i];
    }
    int valMAX = tab[0];
    for (int i = 1; i < n; i++){
        if (valMAX < tab[i])
            valMAX = tab[i];
    }
    printf("La valeur minimal du tableau est : %d\n",valMIN);
    printf("La valeur maximale du tableau est : %d\n",valMAX);
    return 0;
}